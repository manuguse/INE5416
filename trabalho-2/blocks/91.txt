1 2 7 7 12 16 16 16
2 2 7 7 13 13 13 13
2 2 2 9 9 13 13 19
3 3 3 10 14 14 13 19
4 4 3 10 10 15 17 17
4 5 5 11 15 15 18 17
4 6 6 11 15 15 15 17
4 4 8 8 8 8 8 17