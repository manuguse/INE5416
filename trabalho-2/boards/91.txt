- 6 2 - - - 1 2
4 3 - - 7 - - -
- 1 - - - - - -
- - - - 1 - 4 -
- - 3 - 2 5 3 -
4 - 1 - - 3 - -
- - - - - - 6 -
2 5 - 3 1 2 - 1