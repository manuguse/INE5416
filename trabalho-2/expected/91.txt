1 6 2 4 1 3 1 2
4 3 1 3 7 2 6 3
2 1 5 1 2 1 5 2
1 2 4 3 1 2 4 1
6 1 3 1 2 5 3 5
4 2 1 2 4 3 1 4
3 1 2 1 2 1 6 2
2 5 4 3 1 2 5 1